lib/online_order.rb

```diff
@@ -0,0 +1,64 @@
+require_relative "order.rb"
+require_relative "customer.rb"
+module Grocery
+	class OnlineOrder < Grocery::Order
+		attr_reader :order, :customer, :status, :id, :products
+
+		def initialize(id, products, customer, status = "pending")
+			@id = id
+			@products = products
+			@customer = customer
+			@status = status.to_sym
+			@order = super(id, products)
```

Comment:
Two observations here:

1. Because the `initialize` method on `Order` already saves the ID and list of products, you don't need to save them explicitly here - the call to `super` will do it for you.
1. I'm curious why you save the result of `super` as `@order`. You don't seem to use that value anywhere.